dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue

```diff
@@ -173,19 +264,77 @@ const aboutIntroSection = computed(() => {
     outline: none;
     padding-left: 40px;
     background: transparent;
+    font-weight: 600;
     &::placeholder {
       font-size: 16px;
       font-style: normal;
-      font-weight: 600;
-      line-height: 24px; /* 150% */
+
+      line-height: 24px;
       letter-spacing: 0.25px;
-      color: var(--grey-tones-400);
+      color: var(--grey-tones-600);
+    }
+    &:focus {
+      box-shadow: none !important;
+      &::placeholder {
+        color: var(--grey-tones-300);
+      }
+    }
+  }
+
+  &__autocomplete {
```

Comment:
right now this styling is just valid for the CompaniesOnlySearchBar in the newLandingPage. Lets have a quick call if we also want this for the other occurrences of the CompaniesOnlySearchBar.

Reply:
Yes, that would be ideal but in this ticket I have no more time. Please open a new ticket with all these optimisations.

Reply:
okay